chart/values.staging.yaml

```
@@ -19,31 +19,83 @@ analyzeIngress:
 config:
   kubernetes_jobs:
     novostoic-dgpredictor:
-      # TODO: Temporarily run on GPU node
-      nodeSelector:
-        nvidia.com/gpu.present: "true"
-      tolerations:
-        - effect: NoSchedule
-          key: nvidia.com/gpu
-          operator: Exists
-    novostoic-pathways:
-      # TODO: Temporarily run on GPU node
+      nodeSelector:	
+        ncsa.role: worker-job	
+      tolerations:	
+      - effect: NoSchedule	
+        key: mmli.role	
+        operator: Exists
+    novostoic-enzrank:
+      resources:
+        limits:
+          cpu: "1"
+          memory: 12Gi
+        requests:
+          cpu: "1"
+          memory: 8Gi
       nodeSelector:
         nvidia.com/gpu.present: "true"
       tolerations:
         - effect: NoSchedule
           key: nvidia.com/gpu
           operator: Exists
     novostoic-optstoic:
-      # TODO: Temporarily run on GPU node
-      nodeSelector:
-        nvidia.com/gpu.present: "true"
-      tolerations:
-        - effect: NoSchedule
-          key: nvidia.com/gpu
-          operator: Exists
-    # Run on GPU node
+      resources:
+        limits:
+          cpu: "1"
+          memory: 12Gi
+        requests:
+          cpu: "1"
+          memory: 8Gi
+      nodeSelector:	
+        ncsa.role: worker-job	
+      tolerations:	
+      - effect: NoSchedule	
+        key: mmli.role	
+        operator: Exists
+    novostoic-pathways:
+      resources:
+        limits:
+          cpu: "1"
+          memory: 12Gi
+        requests:
+          cpu: "1"
+          memory: 8Gi
+      nodeSelector:  
+        ncsa.role: worker-job  
+      tolerations:  
+      - effect: NoSchedule  
+        key: mmli.role  
+        operator: Exists
+    clean:
+      nodeSelector:  
+        ncsa.role: worker-job  
+      tolerations:  
+      - effect: NoSchedule  
+        key: mmli.role  
+        operator: Exists
+    molli:
+      nodeSelector:  
+        ncsa.role: worker-job  
+      tolerations:  
+      - effect: NoSchedule  
+        key: mmli.role  
+        operator: Exists
+    aceretro:
+      nodeSelector:  
+        ncsa.role: worker-job  
+      tolerations:  
+      - effect: NoSchedule  
+        key: mmli.role  
+        operator: Exists
```

@ckouder hmmm there was no indication in the configuration that ACERetro was using the GPU, but I suppose that makes sense 🤔 we can run ACERetro on the GPU node as well using the following: 
```suggestion
    aceretro:
      nodeSelector:
        nvidia.com/gpu.present: "true"
      tolerations:
        - effect: NoSchedule
          key: nvidia.com/gpu
          operator: Exists
```